### ISSUE TITLE
The v0.8.3 Tags is pointing to the v0.8.2 commit

### ISSUE THREAD

**Issue by ant31** [kubernetes-sigs/application#195]
The tag v0.8.3 is identical to the v0.8.2
```
commit 382977a79c8a57f21c686b7b32a364165c668f87 (HEAD, tag: v0.8.3, tag: v0.8.2, origin/release-v0.8)
Author: Barni S <barni@google.com>
Date:   Mon Apr 13 14:34:46 2020 -0400

    bump to 0.8.2 to in release-0.8 branch
```

**Comment by nan-yu** [comment]
The tag v0.8.3 was updated to the latest commit. However, there is an issue in the release script: https://github.com/kubernetes-sigs/application/blob/master/Makefile#L328
Currently, it is set to release from a `release-v0.8` branch. However, the latest changes were not cherry-picked to the `release-v0.8` branch. I ended up releasing from the master branch without the script.

We need to further enhance the release script when we support multiple major versions in the future.
